cldf/forms.csv

```diff
@@ -866,7 +855,8 @@ Dikhyang-Lieberherr-2017-100-58-1,,Dikhyang,Lieberherr-2017-100-58,siam,siam,s i
 Dikhyang-Lieberherr-2017-100-59-1,,Dikhyang,Lieberherr-2017-100-59,ə-bɛŋ,ə-bɛŋ,ə + b ɛ ŋ,,Lieberherr2017,11701,
 Dikhyang-Lieberherr-2017-100-60-1,,Dikhyang,Lieberherr-2017-100-60,bu-rui,bu-rui,b u + r ui,,Lieberherr2017,11901,
 Dikhyang-Lieberherr-2017-100-61-1,,Dikhyang,Lieberherr-2017-100-61,raguŋ ju-ŋã,raguŋ ju-ŋã,r a g u ŋ + j u + ŋ ã,,Lieberherr2017,12103,
-Dikhyang-Lieberherr-2017-100-62-1,,Dikhyang,Lieberherr-2017-100-62,ə-vɔ̃,ə-vɔ̃,ə + v ɔ̃,,Lieberherr2017,12302,
+Dikhyang-Lieberherr-2017-100-62-1,,Dikhyang,Lieberherr-2017-100-62,"ə-vɔ̃
+",ə-vɔ̃,ə + v ɔ̃,,Lieberherr2017,12302,
```

Comment:
something weird here?

Reply:
Weird indeed, I must have missed something when building the FormSpec.

Reply:
The issue was that, in previous version, the replacement was performed on the `value` -- `FormSpec` only takes care, as expected, of `form`s.